src/libumf.map

```diff
@@ -117,3 +114,9 @@ UMF_1.0 {
     local:
         *;
 };
+
+UMF_0.11 {
+        umfFixedMemoryProviderOps;
+        umfFixedMemoryProviderParamsCreate;
+        umfFixedMemoryProviderParamsDestroy;
+} UMF_0.10;
```

Comment:
not sure on this one - I based this on info from https://sourceware.org/binutils/docs/ld/VERSION.html